TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -0,0 +1,423 @@
+
+**Property name:** `acrylicOpacity`
+
+**Necessity:** Optional
+
+**Accepts:** Number as a floating point value from 0-1
+
+**Default value:** `0.5`
+
+<br />
+
+___
+
+## Background image settings
+
+More detailed information can be found on the [background images page](./../background-images.md).
+
+### Setting the background image
+
+This sets the file location of the image to draw over the window background.
+
+**Property name:** `backgroundImage`
+
+**Necessity:** Optional
+
+**Accepts:** File location as a string
+
+### Background image stretch mode
+
+You can set how the background image is resized to fill the window.
+
+**Property name:** `backgroundImageStretchMode`
+
+**Necessity:** Optional
+
+**Accepts:** `"none"`, `"fill"`, `"uniform"`, `"uniformToFill"`
+
+**Default value:** `"uniformToFill"`
+
+### Background image alignment
+
+This sets how the background image aligns to the boundaries of the window.
+
+**Property name:** `backgroundImageAlignment`
+
+**Necessity:** Optional
+
+**Accepts:** `"center"`, `"left"`, `"top"`, `"right"`, `"bottom"`, `"topLeft"`, `"topRight"`, `"bottomLeft"`, `"bottomRight"`
+
+**Default value:** `"center"`
+
+### Background image opacity
+
+You can set the transparency of the background image.
+
+**Property name:** `backgroundImageOpacity`
+
+**Necessity:** Optional
+
+**Accepts:** Number as a floating point value from 0-1
+
+**Default value:** `1.0`
+
+<br />
+
+___
+
+## Scroll settings
+
+### Scrollbar visibility
+
+Set the visibility of the scrollbar.
+
+**Property name:** `scrollbarState`
+
+**Necessity:** Optional
+
+**Accepts:** `"visible"`, `"hidden"`
+
+### Scroll to input line when typing
+
+When set to `true`, the window will scroll to the command input line when typing. When set to `false`, the window will not scroll when you start typing.
+
+**Property name:** `snapOnInput`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `true`
+
+**Default value:** `true`
+
+### History size
+
+Set the number of lines above the ones displayed in the window you can scroll back to.
+
+**Property name:** `historySize`
+
+**Necessity:** Optional
+
+**Accepts:** Integer
+
+**Default value:** `9001`
+
+<br />
+
+___
+
+## How the profile closes when exiting
+
+Set how the profile reacts to termination or failure to launch. `"graceful"` will close the profile when `exit` is typed or when the process exits normally. `"always"` will always close the profile and `"never"` will never close the profile. `true` and `false` are accepted as synonyms for `"graceful"` and `"never"` respectively.
+
+**Property name:** `closeOnExit`
+
+**Necessity:** Optional
+
+**Accepts:** `"graceful"`, `"always"`, `"never"`, `true`, `false`
+
+**Default value:** `"graceful"`
+
+<br />
+
+___
+
+## Retro Terminal Effects
+
+When set to `true`, enable retro terminal effects. This is an experimental feature and its continued existence is not guaranteed.
```

Comment:
Should probably describe further what we mean by retro. "Emulates a classic CRT display with scanlines and blurry text edges."